.github/workflows/publish.yml

```diff
@@ -94,7 +94,13 @@ jobs:
           files: |
             release/binaries/*
           body_path: release/release-notes.md
+      - name: Publish to Winget
+        uses: vedantmgoyal9/winget-releaser@main
+        if: ${{ steps.version.outputs.is_stable }}
+        with:
+          identifier: amir1376.ABDownloadManager
+          token: ${{ secrets.WINGET_TOKEN }}
```

Comment:
Can you please move this job to another workflow. because in this workflow I'm creating a draft release . (to be confirmed before publishing)

and this job should be triggered when a new release is actually published